app/models/event_stream.rb

```diff
@@ -43,6 +43,38 @@ def emit_notifications
     _log.log_backtrace(err)
   end
 
+  def self.group_levels(strings, regexps, event_groups)
+    return [strings, regexps] unless event_groups
+
+    event_groups.each do |group, level_values|
+      GROUP_LEVELS.each do |level|
+        level = level.to_sym
+        level_values[level]&.each do |value|
+          if value.kind_of?(Regexp)
+            regexps << [value, [group, level]]
+          elsif value.kind_of?(String)
+            if value[0] == "/"
+              regexps << [Regexp.new(value[1..-2]), [group, level]]
+            else
+              strings[value] = [group, level]
+            end
+          end
+        end
+      end
+    end
+    [strings, regexps]
+  end
+
+  def self.event_groups2
```

Comment:
Need a better method name here 😉

Reply:
And perhaps it should be a private method?

Reply:
removed it